### ISSUE TITLE
Dropdown: test and update dropdown to work better in JAWS

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#4946]
**Describe the issue. What is the expected and unexpected behavior?**
It was recently reported by an OpenShift customer that the dropdown didn't work when using:
Screen reader: **JAWS 2019**
Browser: **Chrome**
Which resulted in the following OpenShift [issue](https://bugzilla.redhat.com/show_bug.cgi?id=1884629). 

**Jenn tested this in JAWS and found:**
The main issues happening with JAWS are:
- Focus stays on the toggle/popup button, and announces the button as collapsed. I think this might be that JAWS announces the state of aria-expanded before we have a chance to update it.
- When Forms mode is not engaged before interacting with the button, the menu items are announced in a confusing way. So the user hears that the button is collapsed, then moves to the next item, the Administrator menu item, but it is announced as “Administrator menu”, and then the next item as “Developer menu”

******For updates to make**, my thinking is to remove aria-haspopup, and see if that improves the experience in JAWS. The aria-haspopup is the attribute that tells JAWS that this is a form component, and it needs to go into Forms mode for allowing the user to interact with it. There are two ways that I know of for enabling Forms mode:
1. use the Tab key to shift focus to the button
2. use the Enter key while focus is on the button

For method 1, the component works as expected, because Forms mode is already enabled before the user hits Enter. For method 2, the component doesn’t work as expected, because the behavior we associate with the Enter key is overwritten by JAWS.

Essentially, JAWS owns all key commands when it’s running. So if you have focus on an element, and say that the down arrow key should jump somewhere, JAWS is going to override that with the behavior that it associates with the Down arrow key, which is to go to the next DOM element. BUT if Forms mode is enabled, that is JAWS acknowledging that in a form, the key commands provided by the form components are more important and therefore stops overriding key commands (i.e. hitting the H key while focus is in an input should enter the character “h”, not jump to the next heading).
So the reason why method 2 is wonky is because Forms mode isn’t enabled yet. And when the user hits the Enter key, JAWS overrides our behavior for the component and instead enters into Forms mode, where we now have freedom to define behavior for key presses.

***I think we could create a testing PR for options, test it in JAWS, and potentially provide it to user to determine the exact ways they are navigating and which option works best.

**For full background, when testing in JAWS her experience was:** 
- When focus is on the toggle and I hit Enter, the button is announced as collapsed, which could be confusing. I am able to use the down arrow key to shift focus, but it announces menu items as “Administrator menu” and “Developer menu”, which could also be confusing. I am able to select the one that has focus and see that the UI behaves as expected.

- Sometimes clicking Space when focus is on the toggle will successfully trigger the focus to move to the menu, but this is difficult to reproduce. When focus is in the menu, I hear “Menu, Administrator button” then “Developer button”.

- If I use the Tab key to shift focus to the component, then JAWS will automatically switch to Forms mode. And then when interacting with the Dropdown, it behaves as expected—focus moves to the menu on open, and the arrow keys let me change my selection, and menu items are announced as “Adminstrator button”.

- Currently Forms mode being enabled is required for this component to work. Upon initial interaction, it will switch to forms mode (due to the presence of aria-haspopup which tells JAWS that this is a form element), but JAWS does not switch to Forms mode in time to make the menu usable. Since this component isn’t in the context of a Form, the likeliness of a JAWS user navigating to the menu using Tab is low.

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Bug
